pages/index.js

```diff
@@ -1,3 +1,108 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+import products from "@/products.json";
+
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <AppContainer>
```

Comment:
im sure you will come back to this later on (i think you had a todo comment) but just case: you may want to create a Layout component, for a better overview. if you need a different layout for other pages you can also create a NestedLayout component and add a property getLayout to your page. [This is the Next.js page im referencing](https://nextjs.org/docs/basic-features/layouts)